src/coreclr/tests/src/JIT/Directed/Convert/ldind_conv.il

```diff
@@ -4,7 +4,7 @@
 
 
 .assembly extern mscorlib { auto }
-.assembly extern System.Private.CoreLib { auto }
+.assembly extern System.Runtime { auto }
 
 .assembly test { }
 
```

Comment:
`.class auto Program extends [System.Private.CoreLib]System.Object` below needs to be changed to `System.Runtime` as well. (Multiple places.)

Reply:
How to deal with this `call void [System.Private.CoreLib]Internal.Console::WriteLine`?

Reply:
Change it to `[System.Console]System.Console.WriteLine` or `[mscorlib]System.Console.WriteLine`